src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,11 @@
+package seedu.addressbook.data.person;
+
+
+ /* An interface for printable string represation of a contact's details
+*/
+
+
+public interface Printable {
+
+    String getPrintableString() ;
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.